apps/betterangels-backend/tasks/models.py

```diff
@@ -0,0 +1,34 @@
+import pghistory
+from accounts.models import User
+from common.enums import SelahTeamEnum
+from common.models import BaseModel
+from django.db import models
+from django_choices_field import TextChoicesField
+from organizations.models import Organization
+from tasks.enums import TaskStatusEnum
+
+
+@pghistory.track(
+    pghistory.InsertEvent("task.add"),
+    pghistory.UpdateEvent("task.update"),
+    pghistory.DeleteEvent("task.remove"),
+)
+class Task(BaseModel):
+    client_profile = models.ForeignKey(
+        "clients.ClientProfile", on_delete=models.SET_NULL, blank=True, null=True, related_name="tasks"
+    )
+    created_by = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, related_name="tasks")
+    description = models.TextField(blank=True, null=True)
+    note = models.ForeignKey("notes.Note", on_delete=models.SET_NULL, blank=True, null=True, related_name="tasks")
+    organization = models.ForeignKey(Organization, on_delete=models.SET_NULL, null=True, related_name="tasks")
+    status = TextChoicesField(choices_enum=TaskStatusEnum)
+    summary = models.CharField(max_length=100)
+    team = TextChoicesField(SelahTeamEnum, null=True, blank=True)
```

Comment:
Currently, I don’t see any database indexes on these columns. We should add individual indexes... or even a composite index on the fields we query most frequently to improve performance.